include/common.hpp

```diff
@@ -72,6 +72,11 @@ namespace common
 
     static constexpr name GROUP_TYPE_OPTION = name("option");
 
+    constexpr name LAST_TIME_SHARE = name("lastimeshare");
+    constexpr name CURRENT_TIME_SHARE = name("curntimeshare");
```

Comment:
IMO it should be `curtimeshare`, but not sure if current is abbreviated as `curn` in other places - In that case it would be fine.

I [googled](https://www.allacronyms.com/current/abbreviated) a bit and current is usually abbreviated as  `cur`, `curr` or `curt` - I prefer the first.

Reply:
Yeah, I think it's a better alternative. Didn't spend much time on the naming tbh 😄